style.css

```diff
@@ -0,0 +1,277 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: sans-serif;
+}
+
+*::before, *::after {
+  box-sizing: inherit;
+}
+
+.boss {
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  background-image: url(http://gaslampden.com/wp-content/uploads/sites/3/2014/04/surfer-1.jpg);
+
+  background-size: 100% 100%;
+  background-position: center;
+  position: relative;
+  width: 100%;
+  display: flex;
```

Comment:
Only resort to flexbox when needed. Here you have content just stacking on each other.

Flexbox limits and changes some of the stuff you can do with margins. removing this will make it a bit easier to achieve your styles.

Remember that plain text items can center their text using `text-align: center`